cvmfs/download.h

```diff
@@ -138,6 +145,11 @@ struct JobInfo {
     probe_hosts = false;
     head_request = false;
     follow_redirects = false;
+    pid = -1;
```

Comment:
I wonder if this should be pushed down to the download manager.  Another option would be to attach a string with the credentials to the download job.